src/js/bookmarkList.js

```diff
@@ -1,15 +1,11 @@
-var _ = require('../vendor/underscore/underscore');
```

Comment:
One nice thing about moving to npm is that we can require the packages without the full path. I also removed a bunch of stuff since it doesn't seem to me like it is being used anywhere.